messages/EventKeyTemplates.js

```diff
@@ -8,8 +8,9 @@ module.exports = {
 		"invalidCategorySkipped": "Cataloger skipped invalid PMD Category file '%s'.",
 		"invalidRulesetSkipped": "Cataloger skipped invalid PMD Ruleset file '%s'.",
 		"xmlDropped": "Dropping XML file [%s] since its path does not conform to Rulesets or Category.",
-		"pmdSkippedFile": "PMD failed to evaluate against file '%s'. Message: %s"
-	},
+		"pmdSkippedFile": "PMD failed to evaluate against file '%s'. Message: %s",
+		"catalogFileNotFound": "Catalog file [%s] for language [%s] was not found.",
```

Comment:
I wasn't sure if `Catalog` file is the correct term for this. It could be a jar or xml file that has been moved or deleted.

Reply:
Not sure if a user would understand what "catalog" is. How about "file path" so that we don't worry if this is an XML or JAR?

Reply:
I agree with @rmohan20 that we should avoid using `catalog` in our messages. Perhaps `Custom rule path`?